.bcr/metadata.template.json

```diff
@@ -0,0 +1,15 @@
+{
+  "homepage": "https://github.com/aranguyen/rules_platform",
```

Comment:
This should be rules_platform, not aranguyen, right?

Reply:
oh good catch. This should be https://github.com/bazelbuild/rules_platform. I'll update in the morning